src/Resources/app/administration/src/module/sw-in-app-purchases/component/sw-in-app-purchase-checkout/index.ts

```diff
@@ -0,0 +1,164 @@
+import type * as IAP from 'src/module/sw-in-app-purchases/types';
+import template from './sw-in-app-purchase-checkout.html.twig';
+import './sw-in-app-purchase-checkout.scss';
+
+interface ErrorResponse {
+    errors: Array<{
+        status: string;
+        code: string;
+        title: string;
+        detail: string;
+        meta: {
+            parameters: string[];
+        };
+        trace: Array<{
+            file: string;
+            line: number;
+            function: string;
+            class: string;
+            type: string;
+            args?: string[];
+        }>;
+    }>;
+}
```

Comment:
Just copy what you need
```suggestion
interface ErrorResponse {
    errors: Array<ShopwareHttpError>;
}
```